integrationtests/test_all_examples.py

```diff
@@ -197,16 +197,6 @@ def get_python_files(self):
     lambda: not is_wx(),
     "Only support wx",
 )
-SEARCHER.skip_file_if(
-    os.path.join(DEMO, "Misc", "demo_group_size.py"),
-    is_qt6,
-    "enable tries to import a missing constant. See enthought/enable#891",
-)
```

Comment:
Is this a drive-by fix?

Reply:
Yes, it's a drive-by fix now that Enable understands Qt6.